### ISSUE TITLE
Comprehensive guide for dual-write workflows

### ISSUE THREAD

**Issue by dsieczko** [authzed/docs#265]
We're missing a comprehensive implementation guide that shows how someone can implement various relationship write patterns:

e.g. dual write with workflow engines like Temporal and CDC patterns with Debezium

**Comment by tstirrat15** [comment]
I'm planning on starting with a blog post to this effect and then moving it into documentation.

**Comment by tstirrat15** [comment]
Yeah that's fair. They should be searchable through the same text box too... Let's table this for the time being.
